ohsome_quality_analyst/indicators/currentness/indicator.py

```diff
@@ -229,11 +230,15 @@ def create_figure(self):
         fig.update_yaxes(
             title_text="Percentage of Latest Contributions",
             tickformat=".0%",
+            color="#444",  # #444 is default color
+            gridcolor="#444",  # #444 is NOT default color
         )
         fig.update_yaxes(
             title_text="Absolute Number of Latest Contributions",
             tickformat=".",
             secondary_y=True,
+            color="#000080",
```

Comment:
This is where the color is defined. Does this color suit or should another be used?